src/server/api/controllers/coordinates.controller.js

```diff
@@ -0,0 +1,46 @@
+const knex = require('../../config/db');
+
+const getCoordinates = async () => {
+    try {
+        return await knex('maps').select(
+            'maps.lat_top_left',
+            'maps.long_top_left',
+            'maps.lat_top_right',
+            'maps.long_top_right',
+            'maps.lat_bottom_left',
+            'maps.long_bottom_left',
+            'maps.lat_bottom_right',
+            'maps.lat_bottom_right',
+        );
+    } catch (error) {
+        return error.message;
+    }
+};
+
+const getCoordinatesById = async (id) => {
+    try {
+        const coordinates = await knex('maps')
```

Comment:
Can you fix the duplication here, please

Reply:
@senner008 i didnt get you. Could you please tell me which is the duplication? one is to get all the coordinates and another is to get coordinates by id. The actual issue is to get all the coordinates from map #92 . So is it also necessary to get coordinates by id?

Reply:
@krithi-kondana 
Seems to me that you have already fixed it